_data/simple-icons.json

```
@@ -3410,6 +3410,11 @@
             "hex": "F43E37",
             "source": "https://blog.pocketcasts.com/press/"
         },
+        {
+            "title": "Pokémon GO",
+            "hex": "FF0000",
+            "source": "https://www.pokemongo.com/"
```

Given that the SVG now comes from somewhere else, can you update the source @KovuTheHusky?

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Pokemon_Go.png"
```